tutoring.md

```diff
@@ -0,0 +1,57 @@
+# Ada's Tutoring Services
+
+## Table of Contents
+1. [Overview](#overview)
+1. [Services](#services)
+  1. [Open Tutoring](#open-tutoring)
+  1. [Ad-hoc Tutoring](#ad-hoc-tutoring)
+  1. [Secheduled Tutoring](#scheduled-tutoring)
+1. [FAQ](#faq)
+  1. [Which tutoring service should I use?](#which-tutoring-service-should-i-use)
+
+## Overview
+Ada Developers Academy is committed to ensuring that all of our students receive the support they need to succeed with our curriculum and go on to become professional software developers.
+
+To that end we offer a number avenues for accessing tutoring services outside of class hours. This document explains the different tutoring services offered including when and where they are available and has a FAQ for guiding you through the process of selecting an appropriate tutoring service.
+
+## Services
+### Open Tutoring
+Open Tutoring is our most available tutoring service and should likely be your first choice if you feel tutoring is necessary.
+
+Open Tutoring consists of a fixed set of hours during which experienced software developers are available at the Ada campus to answer questions and assist with debugging for any students present at that time.
+
+Due to the nature of Open Tutoring there aren't always enough Tutors for each student to receive 1-on-1 dedicated time through the entire session, however our Tutors attempt to make sure they work with every student at least once.
+
+The current schedule for Open Tutoring is:
+
+| Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday |
+|:------:|:-------:|:---------:|:--------:|:------:|:--------:|:------:|
+|        | 5-7 PM  |           | 5-7 PM   |        | 11 - 2PM |        |
+
+(Last updated 2/23/2016 - Check your class Google calendar for temporary schedule changes.)
+
+### Ad-hoc Tutoring
```

Comment:
Also - just FYI Brooks will end his sessions at the end of Ruby - so that's pretty soon. Maybe we can see if someone else with Rails knowledge (maybe a previous Adie) would be willing to do a Saturday session?